addon-test-support/test-support/sinon-sandbox.js

```diff
@@ -0,0 +1 @@
+export * from '../sinon-sandbox';
```

Comment:
This exists so that I don't have to expose it on the main import path but can import it into my tests. If you have a better suggestion for the long term, I'm open to that.

Reply:
This seems fine for now. Let's leave it and revisit later if needed.